### ISSUE TITLE
Do you have a special licence or something ?

### ISSUE THREAD

**Issue by maifeeulasad** [delimce/recorderTest#1]
Here is the error message :

`Caused by: java.lang.RuntimeException: JNIWrapper license check failed: No valid license found. jar:file:/C:/Users/MUA/Downloads/recorderTest-master/libs/license.jar!/META-INF/teamdev.licenses - Sorry! Your 30-day evaluation expired on 25-01-2018. 
If you'd like to continue using JNIWrapper in your project, check this out: https://www.teamdev.com/jniwrapper#licensing-pricing
If you feel that you need to evaluate JNIWrapper for a bit longer before purchasing it, you are welcome to fill in our Extended Evaluation form: https://goo.gl/1Y7H9o
Exception running application com.userlytics.recorderTest.Main`

**Comment by delimce** [comment]
No, I don't but if I would need to use this lib, I would buy it
